neonvm/tools/vm-builder/files/sshd_config

```diff
@@ -85,7 +85,7 @@ X11Forwarding no
 #Banner none
 
 # override default of no subsystems
-Subsystem	sftp	/usr/lib/ssh/sftp-server
+Subsystem	sftp	internal-sftp
```

Comment:
Could you please clarify in commit message what is the root cause of the issue, and how does this change fixes it?

Reply:
I added comment which explains what is internal-sftp and why it is in use. Also, we don't have `/usr/lib/ssh/sftp-server` binary in the guest VM, but I ain't sure if that should be mentioned in the pr

Reply:
My 2 cents: good to mention in the PR ("it didn't work before because ..."), but comment is great as you've written.

Reply:
I updated the PR description to have more context in it